external/vcm/vcm/derived_mapping.py

```diff
@@ -360,3 +360,43 @@ def water_vapor_path(self):
     return da.assign_attrs(
         {"long_name": "column integrated water vapor", "units": "mm"}
     )
+
+
+@DerivedMapping.register(
+    "total_sky_net_radiative_flux_at_top_of_atmosphere",
+    required_inputs=[
+        "total_sky_downward_shortwave_flux_at_top_of_atmosphere",
+        "total_sky_upward_shortwave_flux_at_top_of_atmosphere",
+        "total_sky_upward_longwave_flux_at_top_of_atmosphere",
+    ],
+)
+def total_sky_net_radiative_flux_at_top_of_atmosphere(self):
```

Comment:
Is "sky" implied by "top of atmosphere"? Or is this some CF name that you are using?
```suggestion
def total_net_radiative_flux_at_top_of_atmosphere(self):
```

Reply:
"total sky" is in contrast with "clear sky" (i.e. no clouds). I'm following convention established here: 

https://github.com/ai2cm/fv3gfs-fortran/blob/cf1eb54ab439059dc7796e3b0b16404a5dda4966/FV3/wrapper/fv3gfs/wrapper/physics_properties.json#L52